cli/tools/fmt.rs

```diff
@@ -63,8 +61,30 @@ pub async fn format(
   Ok(())
 }
 
+/// Formats markdown (using https://github.com/dprint/dprint-plugin-markdown) and its code blocks
+/// (ts/tsx, js/jsx).
+fn format_markdown(file_text: &str) -> Result<String, String> {
+  let ts_config = get_typescript_config();
+  let md_config = get_markdown_config();
+  dprint_plugin_markdown::format_text(
+    &file_text,
+    &md_config,
+    Box::new(move |tag, text, _| {
```

Comment:
Sorry, the docs do not explain this, but you may want to format the TypeScript code using the `line_width` provided to this closure (the third parameter), as it may contain the line width to use to format the code block.

For example, a code block could be a couple levels deep in the markdown and instead of using 80 characters of width, it should now use 72.

```ts
Box::new(move |tag, text, line_width| {
    ts_config.line_width = line_width;
    // etc...
})
```

...but that's only if you want this behaviour.